### PR TITLE
More countries, fake city and venue names for seeds.

### PR DESCRIPTION
**Pull request by pedrosino**
Makes our local databases more fun.

### CODE REVIEW

**Comment by jfly** [comment]
Good idea! How do you feel about randomly grabbing countries from Country.rb?

**Comment by pedrosino** [comment]
That is probably a good idea. Let me see how to...

**Comment by pedrosino** [comment]
Found it: Country::ALL_COUNTRIES.map(&:id).sample

countries

Just sent a new commit.
I'm not 100% satisfied with the venues names' length, but I can live with that :)

(Maybe Faker::Commerce.product_name #=> "Practical Granite Shirt")

File: WcaOnRails/db/seeds/development/competitions.seeds.rb [hunk 1]
```diff
@@ -13,13 +11,13 @@
       id: "My#{i}Comp#{day.year}",
       name: "My #{i} Best Comp #{day.year}",
       cellName: "My #{i} Comp #{day.year}",
-      cityName: "San Francisco",
-      countryId: countries.sample,
+      cityName: Faker::Address.city,
+      countryId: Country::ALL_COUNTRIES.map(&:id).sample,
```
Comment (jfly) [inline WcaOnRails/db/seeds/development/competitions.seeds.rb]: I think it's more efficient to sample and then just grab the id.

**Comment by jfly** [comment]
LGTM!

**Pull request merged**